lib/test.js

```diff
@@ -2,6 +2,14 @@
 var isGenerator = require('is-generator').fn;
 var Promise = require('bluebird');
 var setImmediate = require('set-immediate-shim');
+var powerAssert = require('power-assert').customize({
+	output: {
+		renderers: [
+			require('power-assert-renderers').AssertionRenderer,
+			require('power-assert-renderers').SuccinctRenderer
+		]
+	}
+});
```

Comment:
@sindresorhus power-assert architecture consists of two parts. Instrumentation (i.e. transformation) side and runtime side. This code is a runtime side. The diff you mentioned is an instrumentation side. Output rendering belongs to runtime side.